shared/js/ui/views/site.es6.js

```diff
@@ -17,6 +17,7 @@ function Site (ops) {
 
     // cache 'body' selector
     this.$body = window.$('body')
+    this.$protectionRow = window.$('body')
```

Comment:
This assignment seems to be redundant, `this.$protectionRow` is identical to `this.$body`